docusaurus.config.js

```diff
@@ -6,15 +6,15 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: 'Unv language',
-  tagline: 'Universal programming language',
+  title: 'The Unv language',
+  tagline: 'The Universal programming language',
   url: 'https://unv.vercel.app',
   baseUrl: '/',
   onBrokenLinks: 'warn',
   onBrokenMarkdownLinks: 'warn',
   favicon: 'img/favicon.ico',
-  organizationName: 'unvlabs',
-  projectName: 'site',
+  organizationName: 'Unvlabs',
+  projectName: 'The UNV site',
```

Comment:
```suggestion
  projectName: 'site',
```
This should match the name of Github repo contains the site.
```
https://github.com/UnvLabs/site
site is taken from this url
```